### PR TITLE
Add Temps

### PR DESCRIPTION
**Pull request by jackd248**
I want to add my weather app "temps" to the open source list.
[Repo](https://github.com/jackd248/temps) / [Website](https://jackd248.github.io/temps/)

### CODE REVIEW

**Comment by zeke** [comment]
It's not working for me. I put in my city and API key but they don't get saved. When I click [Close], the application quits.

screen shot 2016-09-28 at 10 52 28 am

**Comment by jalcine** [comment]
Yeah, it doesn't even launch for me. This app might require more testing.

**Comment by jackd248** [comment]
@zeke 
Doesn't look like an api key in your screenshot. ;) 
First of all: "Close" is for app quiting at all. If you will go back to weather screen use the settings icon in the upper left or use the shortcut <kbd>cmd</kbd> + <kbd>s</kbd>. I'm sorry, didn't noticed so far that this could be confusing. 
For applying a city or also an api key, you only need to enter them into the field and press <kbd>enter</kbd>. Maybe this could work for you. :)

@jalcine 
I'm really sorry about the troubles and I will include more testing soon. So maybe you can help me figuring out what problems happend with your version. Which platform you are using?

**Comment by zeke** [comment]
> Doesn't look like an api key in your screenshot. ;) 

I took it out. Don't want to share API keys with the world.

> enter them into the field and press enter. 

This is not really intuitive. A button to save would make more sense, or some visual indication that changes are saved.

**Comment by jackd248** [comment]
But the more interesting question: did it worked for you? 
Yeah, I'm thinking about some kind of indicator for saving the api key. I tried to keep the app at least as minimalistic as possible, so don't want to add another button. 
What do you think about the rest of the app? Or is this such a huge negative thing for you?

**Comment by vutran** [comment]
I feel that a generic API key should be baked into the app for general usage. If you are planning to deploy this into the Apple store, users will be turned down to register for an API key to get it working. For everyone else on Github, most of us have no problem obtaining a key.

**Comment by jackd248** [comment]
You are right, it could be a good idea to place a key in the app for the start. But for now, I'm not planning to release or sign the app. So the main target group are open source friends. :)

**Comment by stefanjudis** [comment]
> Yeah, it doesn't even launch for me. This app might require more testing.

same here.

**Comment by jackd248** [comment]
@stefanjudis Which platform?

**Comment by stefanjudis** [comment]
@jackd248 Okay my mistake.

From this landing page https://jackd248.github.io/temps/ it was not clear to me that it's a tray app. 💃 

So it' starting.

File: readme.md [hunk 1]
```diff
@@ -148,6 +148,7 @@ Made with Electron.
 - [Wire](https://github.com/wireapp/wire-desktop) - Messenger and calling app.
 - [Ramme](https://github.com/terkelg/ramme) - Unofficial Instagram app.
 - [Moonview](https://github.com/teesloane/moonview) - Ambient notepad.
+- [Temps](https://jackd248.github.io/temps/) - A simple but smart weather app.
```
Comment (sindresorhus) [inline readme.md]: How is it "smart"?

**Comment by MarshallOfSound** [comment]
I don't think it should really be up to the user to provide an API key.  I also stuggled to make it work (same issue @zeke had I think).  Had to click in and out of the API key box a few times before it appeared to do anything.

**Comment by jackd248** [comment]
I considered your suggestions and release a new version v0.5.1 with some usability improvements and an "starter" api key within the app. Hope this will help everyone getting a better start with Temps ;)

File: readme.md (hunk 1 above)
Comment (jackd248) [inline readme.md]: "Smart" means for me and for Temps, that it is innovative, modern and informative but not intrusive or overloaded. If this is not exact enough for you, "simple weather app" is also working for me.

Comment (sindresorhus) [inline readme.md]: Yeah, that's not what I would consider "smart".

Comment (jackd248) [inline readme.md]: Changed description

**Pull request merged**